db/migrate/20231222201514_create_users.rb

```diff
@@ -0,0 +1,11 @@
+class CreateUsers < ActiveRecord::Migration[7.1]
+  def change
+    create_table :users do |t|
+      t.string :name
+      t.string :photo
+      t.text :bio
+      t.integer :post_counter, default: 0
```

Comment:
- Hi, in your post model, you have used posts_counter, but it is singular here. kindly update it.

Reply:
**[OPTIONAL]**



- Let's try to add the node_mofules/ to the .gitignore file before we make a commit to prevent pushing them to your repo.